driver-scala/src/main/scala/org/mongodb/scala/model/Windows.scala

```diff
@@ -0,0 +1,265 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mongodb.scala.model
+
+import com.mongodb.annotations.Beta
+import com.mongodb.client.model.{ MongoTimeUnit => JMongoTimeUnit, Windows => JWindows }
+import org.bson.types.Decimal128
+import org.mongodb.scala.bson.conversions.Bson
+
+/**
+ * Builders for [[Window windows]] used when expressing [[WindowedComputation windowed computations]].
+ * There are two types of windows: documents and range.
+ *
+ * Bounded and half-bounded windows require sorting.
+ * Window bounds are inclusive and the lower bound must always be less than or equal to the upper bound.
+ * The following type-specific rules are applied to windows:
+ *  - documents
+ *    - bounds
+ *      - 0 refers to the current document and is functionally equivalent to [[Windows.Bound CURRENT]];
+ *      - a negative value refers to documents preceding the current one;
+ *      - a positive value refers to documents following the current one;
+ *  - range
+ *    - [[Aggregates.setWindowFields sortBy]]
+ *      - must contain exactly one field;
+ *      - must specify the ascending sort order;
+ *      - the `sortBy` field must be of either a numeric BSON type
+ *      (see the `\$isNumber` aggregation pipeline stage)
+ *      or the BSON `Date` type if time
+ *      bounds are used;
+ *    - bounds
+ *      - if numeric, i.e., not `com.mongodb.client.model.Windows.Bound`, then the bound is calculated by adding
+ *      the value to the value of the `sortBy` field in the current document;
+ *      - if [[Windows.Bound CURRENT]], then the bound is determined by the current document
+ *      and not the current value of the `sortBy` field;
+ *      - time bounds require specifying a [[MongoTimeUnit time unit]] and are added as per the
+ *      `\$dateAdd`/`\$dateSubtract` aggregation pipeline stage specification.
+ *
+ * @see WindowedComputation
+ * @see [[http://docs.mongodb.org/manual/reference/operator/aggregation/isNumber/ \$isNumber aggregation pipeline stage]]
+ * @see [[http://docs.mongodb.org/manual/reference/bson-types/#date BSON Date type]]
+ * @see [[http://docs.mongodb.org/manual/reference/operator/aggregation/dateAdd/ \$dateAdd aggregation pipeline stage]]
+ * @see [[http://docs.mongodb.org/manual/reference/operator/aggregation/dateSubtract/ \$dateSubtract aggregation pipeline stage]]
+ * @since 4.3
+ * @note Requires MongoDB 5.0 or greater.
+ */
+@Beta
+object Windows {
+
+  /**
+   * Creates a window from [[org.mongodb.scala.bson.conversions.Bson]]
```

Comment:
The Gradle task `:driver-scala:aggregatedScalaDoc` produces the following warning
```
Windows.scala:62: warning: Could not find any member to link for "org.mongodb.scala.bson.conversions.Bson"
```

Links to fully-qualified names are supposed to work according to https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html#markup. @rozza Do you know what could be wrong?

Reply:
Probably because its a type alias and can;t find `org.bson.conversions.Bson`

Reply:
So it is not possible to create a scaladoc link to a program element in a different package?

Reply:
> So it is not possible to create a scaladoc link to a program element in a different package?
Probably but its not going to be a quick fix for today.

Reply:
Replaced with `Bson`. Done.